data/DataType/char_pascal.yaml

```diff
@@ -0,0 +1,6 @@
+keyword: char
+integer-min: '#0'
```

Comment:
Please remove the `#` from here. It is not required.

Reply:
https://www.freepascal.org/docs-html/ref/refsu6.html#x29-320003.2.1 says the char should start with # when ascii code is used....